charts/node-ipam-controller/templates/serviceaccount.yaml

```diff
@@ -32,3 +32,33 @@ subjects:
   - kind: ServiceAccount
     name: {{ include "node-ipam-controller.serviceAccountName" . }}
     namespace: {{ .Release.Namespace }}
+---
+{{- if .Values.leaderElection.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "node-ipam-controller.serviceAccountName" . }}-manager-role
```

Comment:
```suggestion
  name: {{ include "node-ipam-controller.serviceAccountName" . }}-leader-election-role
```

Reply:
Sorry, I was not clear regarding the convention.

Reply:
nope it was my misunderstanding :) should be fixed now